app/views/topics/new.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>New Topic</h1>
+<%= render 'form' %>
+<p class= "padded"><%= link_to "Back" %></p>
```

Comment:
No it does not. I forgot to add the path. I changed it to :

<%= link_to "Back" , topics_path %>

On Fri, Feb 27, 2015 at 8:28 AM, Eliot Sykes <notifications@github.com>
wrote:

> In app/views/topics/new.html.erb
> <https://github.com/LuisMenjivar/luisbookmark/pull/6#discussion_r25505725>
> :
>
> > @@ -0,0 +1,3 @@
> > +<h1>New Topic</h1>
> > +<%= render 'form' %>
> > +<p class= "padded"><%= link_to "Back" %></p>
>
> Does this link work OK?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LuisMenjivar/luisbookmark/pull/6/files#r25505725>.
>